humans/templates/humans/emails/key_will_expire.txt

```diff
@@ -0,0 +1,8 @@
+Hello {% if user.first_name %} {{user.first_name}} {{user.last_name}} {% else %} {{user.username}} {% endif %},
+
+During our automatic updates and checks, using the keyserver url that you provided, the system spotted that the key that is currently present on that URL will soon expire.
```

Comment:
Perhaps we should add the translation blocks to this email, so we could translate it to other languages.